src/components/ec-input-field/ec-input-field.vue

```diff
@@ -135,6 +135,8 @@ const props = withDefaults(defineProps<InputFieldProps>(), {
   isLoading: false,
   isSensitive: false,
   isWarning: false,
+  isInLightMode: false,
+  isDropdownOpen: false,
```

Comment:
Input field shouldn't care about dropdown, we shouldn't introduce props from the context of another component.

Reply:
should i rename it to is-input-bg-white?

Reply:
What is this prop for? Why we didn't have it before? Also could you please add light mode in storybook for dropdown and input and update screenshots.

Reply:
I added it in storybook for dropdown, will add it to input too

Based on the design we have to change the input bg color when the dropdown is visible, that's why i added the prop. We didn't have this change before, that's why we didn't have a prop